phpunit.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit bootstrap="vendor/autoload.php"
+         backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         verbose="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite name="AWS Queue Bundle Test Suite Unit">
+            <directory>Tests/Unit</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist processUncoveredFilesFromWhitelist="true">
```

Comment:
Please could you correct this as it gives a no coverage error as the directory 'src/' does not exist as well as add a blacklist.
I have added the code block below for you to make it easier for us as I already have tested with this :)

```xml
<filter>
        <blacklist>
            <directory suffix=".php">./vendor</directory>
        </blacklist>
        <whitelist processUncoveredFilesFromWhitelist="true">
            <directory suffix=".php">./</directory>
        </whitelist>
</filter>